specification/loadtestservice/LoadTestService/models.trigger.tsp

```diff
@@ -0,0 +1,277 @@
+
+  @doc("Recurrence set to repeat on the specified dates of the month. Value of dates can be 1 to 31 and -1. -1 represents the last day of the month.")
+  datesInMonth?: int32[];
+
+  @doc("The interval at which the recurrence should repeat. It signifies the number of months between each recurrence.")
+  interval?: int32;
+}
+
+@added(APIVersions.v2024_07_01_preview)
+@doc("Recurrence is set based on cron expression.")
+model RecurrenceWithCron extends Recurrence {
+  @doc("Specify frequency using a cron expression.")
+  frequency: Frequency.Cron;
+
+  @doc("Cron expression for the recurrence.")
+  cronExpression: string;
+}
+
+@added(APIVersions.v2024_07_01_preview)
+@doc("Recurrence model when frequency is set as weekly.")
+model WeeklyRecurrence extends Recurrence {
+  @doc("Frequency of the week recurrence.")
+  frequency: Frequency.Weekly;
+
+  @doc("Recurrence set to repeat on the specified days of the week.")
+  daysOfWeek?: WeekDays[];
+
+  @doc("The interval at which the recurrence should repeat. It signifies the number of weeks between each recurrence.")
+  interval?: int32;
+}
+
+@doc("Recurrence end model. Either provide numberOfOccurrences if you want recurrence to end after a specified number of occurrences or provide endDate if you want recurrence to end after a specified end date. If both values are provided, a validation error will be thrown indicating that only one field should be provided. If neither value is provided, the recurrence will end when manually ended.")
+@added(APIVersions.v2024_07_01_preview)
+model RecurrenceEnd {
+  @doc("Number of occurrences after which the recurrence will end.")
+  numberOfOccurrences?: int32;
+
+  @doc("The date after which the recurrence will end. (RFC 3339 literal format)")
+  endDateTime?: utcDateTime;
+}
+
+@added(APIVersions.v2024_07_01_preview)
+@discriminator("frequency")
+@doc("Recurrence model.")
+model Recurrence {
+  @doc("Frequency of the recurrence.")
+  frequency: Frequency;
+
+  @doc("Recurrence end model. You can specify the end either by providing a numberOfOccurrences (which will end the recurrence after the specified number of occurrences) or by providing an endDate (which will end the recurrence after the specified date). If neither value is provided, the recurrence will continue until it is manually ended. However, if both values are provided, an error will be thrown.")
+  recurrenceEnd?: RecurrenceEnd;
+}
+
+@added(APIVersions.v2024_07_01_preview)
+@doc("Parameters for list trigger operation")
+model ListTriggerQueryParams {
+  @doc("Search based on triggers associated with the provided test ids.")
+  @query
+  testIds?: string;
+
+  @doc("Filter triggers based on a comma separated list of states.")
+  @query
+  states?: TriggerState;
+
+  @doc("Start DateTime of trigger creation time filter range. (RFC 3339 literal format)")
+  @query
+  createdFrom?: utcDateTime;
+
+  @doc("End DateTime of trigger creation time filter range. (RFC 3339 literal format)")
+  @query
```

Comment:
@venkatr21/ @Himanshu49 These parameters dont exist. Replace with `lastModifiedStartTime` and `lastModifiedEndTime` . Sorry missed this in review and it is in line with Mike's suggestion

Reply:
Fixed it